lib/ChainwebDb/Types/Transaction.hs

```diff
@@ -88,6 +92,9 @@ Transaction
 type Transaction = TransactionT Identity
 type TransactionId = PrimaryKey TransactionT Identity
 
+instance HasColumnType TsVector where
```

Comment:
I think this might be no longer needed, right?

Reply:
This can be removed.